README.md

```diff
@@ -1,46 +1 @@
-# Getting Started with Create React App
```

Comment:
Try to remove boiler plates and package json file from PR, push them into master branch first

Reply:
Sure will take care from next review